File: WOT-iOS/WOTData/WOTData/CoreData/model/autogenerated/ListSetting+CoreDataProperties.m
//
//  ListSetting+CoreDataProperties.m
//  WOTData
//
//  Created by Pavel Yeshchyk on 1/17/20.
//  Copyright © 2020 Pavel Yeshchyk. All rights reserved.
//
//

#import "ListSetting+CoreDataProperties.h"

@implementation ListSetting (CoreDataProperties)

+ (NSFetchRequest<ListSetting *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"ListSetting"];
}

@dynamic ascending;
@dynamic key;
@dynamic orderBy;
@dynamic type;
@dynamic values;

@end

File: WOT-iOS/WOTData/WOTData/CoreData/model/autogenerated/ListSetting+CoreDataProperties.swift
//
//  ListSetting+CoreDataProperties.swift
//
//
//  Created by Pavel Yeshchyk on 4/23/20.
//
//

import Foundation
import CoreData

extension ListSetting {
    @nonobjc public class func fetchRequest() -> NSFetchRequest<ListSetting> {
        return NSFetchRequest<ListSetting>(entityName: "ListSetting")
    }

    @NSManaged public var ascending: NSNumber?
    @NSManaged public var key: String?
    @NSManaged public var orderBy: NSNumber?
    @NSManaged public var type: String?
    @NSManaged public var values: String?
}
